sdk/lib/drivers/sound/mmixer/mmixer.h

```diff
@@ -207,6 +207,12 @@ MMixerOpenWave(
     IN PVOID Context,
     OUT PHANDLE PinHandle);
 
+MIXER_STATUS
+MMixerGetWavePosition(
+    _In_ PMIXER_CONTEXT MixerContext,
+    _In_ HANDLE PinHandle,
+    _Out_ DWORD * Position);
```

Comment:
```suggestion
    _Out_ PDWORD Position);
```

Reply:
OK, changed it even to `LPDWORD`.

Reply:
well, "even" to LPDWORD actually not using "L" would be better. "L" in these names is a backward compatibility with 16-bit windows where there was a distinction between far and near pointers, and this doesn't make sense anymore since windows being 32-bits. (This is why I suggested "PDWORD" without "L" . Also in pure NT world, no pointer type has this "L" prefix thing.)